README.md

```diff
@@ -492,4 +492,10 @@ $ cd <fdo-pri-src>/component-samples/demo/device
 $ java -jar device.jar
 ```
 
+## Notice: Use of Bouncy Castle FIPS as Security Provider in FDO Project
 
+FDO project utilizes Bouncy Castle FIPS as its primary security provider. This includes all cryptographic operations within the project.
+
+However, it's important to note that there is one exception to this: the Key Derivation Function (KDF) implementation. For this specific operation, we use an unapproved implementation which follows FDO Spec.
```

Comment:
The KDF implementation is compliant with the FIDO specification and is not based on the Bouncy Castle FIPS.